### DISCUSSION TITLE
Interested to contribute

### DISCUSSION THREAD

**Discussion by SagarDollin** [labmlai/annotated_deep_learning_paper_implementations#88] [Show and tell]
Hi @vpj,
I'm [Sagar B Dollin](https://github.com/SagarDollin), Master's Student in [Cognitive Science, Indian Institute of Technology, Delhi](http://cogsci.iitd.ac.in/msc2022/sagar-b-dollin/).

I'm interested in contributing to paper implementations related to deep learning. If you want to request a feature or report a bug feel free to assign those issues to me, and I would like to spend a couple of hours reading the respective paper and try to help resolve the issue.

I have worked on some interesting projects that you can find here:

1. [Computer Vision to perceive Kanizsa Illusion](https://github.com/SagarDollin/Computer_Vision_to_perceive_illusion)
2. [Unsupervised Word Sense Disambiguation on top of word2vec
](https://github.com/SagarDollin/unsupervised_WSD_on_top_of_word2vec)
3. [Computational Models of Motor Imagery](https://github.com/SagarDollin/Computational-Models-of-Motor-Imagery)
4. [More projects](https://github.com/SagarDollin?tab=repositories)

Thank you.

**Comment by vpj** [comment]
Awesome, thanks! Will let you know if something comes up.
